tests/framework/installer/ceph_installer.go

```diff
@@ -579,3 +586,130 @@ func NewCephInstaller(t func() *testing.T, clientset *kubernetes.Clientset, useH
+			}
+			if j.Status.Succeeded == 0 {
+				return false, nil
+			}
+		}
+		return true, nil
+	}
+
+	if err = wait.Poll(5*time.Second, 60*time.Second, allJobsAreComplete); err != nil {
+		return fmt.Errorf("failed to wait for wipe jobs to complete. %+v", err)
+	}
+
+	for _, node := range nodes {
+		// if delete fails, don't worry about the error; delete only on best-effort basis
+		h.k8shelper.Clientset.BatchV1().Jobs(namespace).Delete(wipeJobName(node), &metav1.DeleteOptions{})
+	}
+
+	return nil
+}
+
+// GetDiskWipeJob returns a YAML manifest string for a job which will wipe clean the extra
+// (non-boot) disks on a node, allowing Ceph to use the disk(s) during its testing.
+func (h *CephInstaller) GetDiskWipeJob(nodeName, jobName, namespace string) string {
+	// put the wipe job in the cluster namespace so that logs get picked up in failure conditions
+	return `apiVersion: batch/v1
+kind: Job
+metadata:
+  name: ` + jobName + `
+  namespace: ` + namespace + `
+spec:
+    template:
+      spec:
+          restartPolicy: OnFailure
+          containers:
+              - name: disk-wipe
+                image: ` + h.CephVersion.Image + `
+                securityContext:
+                    privileged: true
+                volumeMounts:
+                    - name: dev
+                      mountPath: /dev
+                    - name: run-udev
+                      mountPath: /run/udev
+                command:
+                    - "sh"
+                    - "-c"
+                    - |
+                      set -xEeuo pipefail
+                      # Wipe LVM data from disks
+                      #
+                      udevadm trigger || true
+                      vgimport -a || true
+                      for vg in $(vgs --noheadings --readonly --separator=' ' -o vg_name); do
+                        lvremove --yes --force "$vg"
+                        vgremove --yes --force "$vg"
+                      done
+                      for pv in $(pvs --noheadings --readonly --separator=' ' -o pv_name); do
+                        pvremove --yes --force "$pv"
+                      done
+                      #
+                      # we CANNOT wipe the boot disk if it exists
+                      fdisk -l # show the output here for helping debug log output
+                      df /
+                      boot_disk=""
+                      # some versions of fdisk identify boot with an asterisk, some with 'boot'
+                      boot_line_cmd="fdisk -l | grep '/dev/' | grep -E '\*|boot'"
+                      if eval "${boot_line_cmd}"; then
+                          boot_disk="$(eval "${boot_line_cmd}" | grep --only-matching --extended-regexp /dev/x?[vs]d[a-z]+)"
+                      fi
+                      # in cloud environments, the disk could possibly be xvd[a-z]
+                      rook_disks="$(find /dev -regex '/dev/x?[vs]d[a-z]+$' -and -not -wholename "${boot_disk}")"
```

Comment:
This cleanup will only work with `/dev/x*`? What if i'm running the tests locally with `/dev/sd*`?

Reply:
The `x?` is to support disks that start with `xvd[a-z]` as mentioned in the comment. The `[vs]` after this will support the `/dev/sd*` disks you're referencing.